templates/repo/view_list.tmpl

```diff
@@ -1,4 +1,4 @@
-<table id="repo-files-table" class="ui single line table gt-mt-0" data-last-commit-loader-url="{{.LastCommitLoaderURL}}">
+<table id="repo-files-table" class="ui single line table gt-mt-0" {{if .HasFilesWithoutLatestCommit}}hx-indicator="tr.notready td.message span" hx-trigger="load" hx-swap="morph" hx-post="{{.LastCommitLoaderURL}}"{{end}}>
```

Comment:
Could there be a delay? eg: 1 second?

Reply:
I can change `hx-trigger="load"` to `hx-trigger="load delay:1s"` but why would you want that?

Reply:
If it reloads too quickly, maybe the cache is still not ready. Then it needs to reload again ....

I think a reasonable delay like 1s could avoid unnecessary reloading.

Reply:
I don't see any case where there is a need for another load... Can you explain how it would happen?

Reply:
For example, what guarantees that GetCommitsInfo would return all ready items? If there is no guarantee, the second loading would still gets some "notready" items.

Reply:
The guarantee is that the timeout is set to 0 so we wait for all the commits, we never cancel

Reply:
Oh thank you. I see, that's really tricky.